Weaponize (#358)
tatianamac/selfdefined: 11ty/definitions/weaponize.md

---
title: weaponize
slug: weaponize
defined: true
speech: verb
excerpt: The act or process of making something capable of causing harm to an individual or group.
---

> The act or process of making something capable of causing harm to an individual or group.

Reviewer: ```suggestion
the act or process of making an idea or entity capable of causing harm to an individual or group.
```

## Issues

> Weaponize can be used as way to accurately highlight the effect of certain words, systems and behaviours that negatively impact vulnerable groups. Unfortunately, it is also frequently wielded by bad faith actors to make accusations that relatively benign or progressive behaviours are causing harm to themselves, when in fact the actions they claim to be 'weaponized' against them are actually vulnerable groups and allies seeking to prevent or disrupt systematic injustices. The word can be, and frequently is itself, weaponized; instead of being a way of framing the context of aggressive action, it becomes part of the defense of someone who themselves is engaging in an act of aggression. Care should be taken when encountering the word used to discern the context in which it is applied before taking it at face value.

Reviewer: The issues section has helpful information in it, but I wonder if we can be a bit less academic in how we write it and break down the ideas. I would recommend splitting this into two sections:

## Usage
(here talk about "legitimate usage"—I think here we need to talk about power.)

## Issues
- (bad faith actors section)
- "Weaponise" as a word itself can be "weaponised" when people in power use it as a defence mechanism against vulnerable groups. ...etc

Some examples might help in an `##Examples` section as well.

## Impact
Identifying a situation where an action, system, or concept is being weaponized, and explicitly identifying it as such, can be a powerful tool in protecting vulnerable sub groups from further harm, by revealing the existence of sometimes subtle oppressions.

> e.g. The reduction of polling locations in certain parts of the country seeks to weaponize the ability to exercise political agency to further disenfranchise the systematically oppressed

Reviewer: ```suggestion
e.g. [WHO IS DOING THIS] reducing the polling locations in certain parts of the United States weaponises the ability to exercise political agency to further disenfranchise the systematically oppressed
```
Do you have other examples that are a bit more straight forward to explain?

---

> title: weaponize

Reviewer: ```suggestion
title: weaponise
```
We use International English so this should be weaponise

> slug: weaponize

Reviewer: ```suggestion
slug: weaponise
```
